File: Email-Server-Client-master/Email-Server-Client-master/src/main/java/com/WebApp/EmailServerClient/Accounts/Account.java
package com.WebApp.EmailServerClient.Accounts;

public abstract class Account {
    public String UserName, Pass, EmailAddress;
    public int ID;

    @Override
    public String toString() {
        return "Account{" +
                "UserName='" + UserName + '\'' +
                ", Pass='" + Pass + '\'' +
                ", ID='" + ID + '\'' +
                ", EmailAddress='" + EmailAddress + '\'' +
                '}';
    }
}

File: Email-Front-main/Email-Front-main/src/app/Account/Account.ts
export class Account {
  public UserName: string;
  public EmailAddress: string;
  public Pass: string;
  public ID: number;

  constructor(UserName: string, EmailAddress: string, Pass: string, ID: number) {
    this.UserName = UserName;
    this.EmailAddress = EmailAddress;
    this.Pass = Pass;
    this.ID = ID;
  }


  setID(value: number) {
    this.ID = value;
  }


  getID(): number {
    return this.ID;
  }
}
